cmd/piper.go

```diff
@@ -47,6 +47,7 @@ type GeneralConfigOptions struct {
 	GCSFolderPath        string
 	GCSBucketId          string
 	GCSSubFolder         string
+	URLReportFileName    string
```

Comment:
let's keep this out of the general configuration. I would suggest to rather handle this per step, e.g. as a dedicated parameter for `shellExecuteScript`